.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 # In order to install Chrome stable we need trusty distribution.
 dist: trusty
 language: python
-
+cache: pip
```

Comment:
This statement is not required. We already cache all third party modules.

Reply:
I am aware of it. Should we follow the same pattern and install these packages to oppia_tools folder too?

Reply:
See comment below.

Reply:
Comment addressed!